examples/hex_variation.py

```diff
@@ -42,7 +41,8 @@ def hex_variation(num_tiles):
     return place_at(diagrams, grid)
 
 
-dia = hex_variation(12).line_width(0.02).rotate_by(1 / 4)
+dia = hex_variation(12).line_width(0.05).show_bounding_box()
```

Comment:
Maybe we can remove the bounding box? But I think I understand why you looked at it: the output is not centered for whatever reason.